database/nakadi/10_data/04_tables/05_timeline.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE zn_data.timeline (
+  t_id              UUID                     NOT NULL PRIMARY KEY,
```

Comment:
@v-stepanov Timeline id is machine generated, and it is better to have something unique while searching thru logs. 
Storage id - is something, that we will use as humans to perform switching, that is why it should be created by human and readable by human(values like default, kafka_10_test or whatever, but human readable)